### DISCUSSION TITLE
Pandora AI for New Bing

### DISCUSSION THREAD

**Discussion by araea** [viopsa233/AI-testers-Forum#9] [技术交流]
<html>
<body>
<div class="lake-content" typography="classic"><details class="lake-collapse"><summary id="u6b86c89f"><strong><span class="ne-text">具体步骤：</span></strong></summary><ol class="ne-ol" style="margin: 0; padding-left: 23px"><li id="u96d0fd39" data-lake-index-type="0"><span class="ne-text">下载“Pandora AI”</span><a href="https://github.com/waylaidwanderer/PandoraAI" data-href="https://github.com/waylaidwanderer/PandoraAI" target="_blank" class="ne-link"><span class="ne-text">https://github.com/waylaidwanderer/PandoraAI</span></a><span class="ne-text">  </span></li></ol><ul class="ne-ul" style="margin: 0; padding-left: 23px"><li id="uee13c1d1" data-lake-index-type="0"><span class="ne-text">Code -&gt; Download ZIP</span></li><li id="u3165aa81" data-lake-index-type="0"><span class="ne-text">解压后得到“PandoraAI-main”文件夹</span></li></ul><ol start="2" class="ne-ol" style="margin: 0; padding-left: 23px"><li id="u3960e4ba" data-lake-index-type="0"><span class="ne-text">安装所需文件</span></li></ol><ul class="ne-ul" style="margin: 0; padding-left: 23px"><li id="u5300f010" data-lake-index-type="0"><span class="ne-text">在“PandoraAI-main”文件目录下打开终端  </span></li><li id="ub975d39f" data-lake-index-type="0"><span class="ne-text">输入指令（三选一）</span><code class="ne-code" style="font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier, monospace; background-color: rgba(0, 0, 0, 0.06); border: 1px solid rgba(0, 0, 0, 0.08); border-radius: 2px; padding: 0px 2px"><span class="ne-text">npm install</span></code><code class="ne-code" style="font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier, monospace; background-color: rgba(0, 0, 0, 0.06); border: 1px solid rgba(0, 0, 0, 0.08); border-radius: 2px; padding: 0px 2px"><span class="ne-text">yarn install</span></code><span class="ne-text"> </span><code class="ne-code" style="font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier, monospace; background-color: rgba(0, 0, 0, 0.06); border: 1px solid rgba(0, 0, 0, 0.08); border-radius: 2px; padding: 0px 2px"><span class="ne-text">pnpm install</span></code></li></ul><div class="ne-quote" style="margin: 5px 0; padding-left: 1em; border-left: 3px solid #eeeeee"><p id="u9d7ab33c" class="ne-p" style="margin: 0; padding: 0; min-height: 24px"><span class="ne-text">如果没有yarn和pnpm 可以使用指令安装</span></p><p id="uba0837d6" class="ne-p" style="margin: 0; padding: 0; min-height: 24px"><code class="ne-code" style="font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier, monospace; background-color: rgba(0, 0, 0, 0.06); border: 1px solid rgba(0, 0, 0, 0.08); border-radius: 2px; padding: 0px 2px"><span class="ne-text">npm install --global yarn</span></code></p><p id="ud6740cd8" class="ne-p" style="margin: 0; padding: 0; min-height: 24px"><code class="ne-code" style="font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier, monospace; background-color: rgba(0, 0, 0, 0.06); border: 1px solid rgba(0, 0, 0, 0.08); border-radius: 2px; padding: 0px 2px"><span class="ne-text">npm install -g @pnpm/exe</span></code></p><p id="uee2a647f" class="ne-p" style="margin: 0; padding: 0; min-height: 24px"><code class="ne-code" style="font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier, monospace; background-color: rgba(0, 0, 0, 0.06); border: 1px solid rgba(0, 0, 0, 0.08); border-radius: 2px; padding: 0px 2px"><span class="ne-text">iwr </span><a href="https://get.pnpm.io/install.ps1" data-href="https://get.pnpm.io/install.ps1" target="_blank" class="ne-link"><span class="ne-text">https://get.pnpm.io/install.ps1</span></a><span class="ne-text"> -useb | iex</span></code></p></div><ul class="ne-ul" style="margin: 0; padding-left: 23px"><li id="u831249e9" data-lake-index-type="0"><span class="ne-text">正常情况不会报错 如果出现错误 请想办法解决 不然无法成功   </span></li></ul><ol start="3" class="ne-ol" style="margin: 0; padding-left: 23px"><li id="uc16c4be0" data-lake-index-type="0"><span class="ne-text">得到“.env”文件</span></li></ol><ul class="ne-ul" style="margin: 0; padding-left: 23px"><li id="u0a2d7a4f" data-lake-index-type="0"><span class="ne-text">在PandoraAI的文件目录中找到“.env.example” -&gt; 复制一份 -&gt; 将其重命名为“.env”  </span></li></ul><ol start="4" class="ne-ol" style="margin: 0; padding-left: 23px"><li id="ucdfe0fd2" data-lake-index-type="0"><span class="ne-text">设置“settings.js”</span></li></ol><ul class="ne-ul" style="margin: 0; padding-left: 23px"><li id="u8187c604" data-lake-index-type="0"><span class="ne-text">“node-chatgpt-api”文件目录打开“settings.js”找到  </span></li></ul><div class="ne-quote" style="margin: 5px 0; padding-left: 1em; border-left: 3px solid #eeeeee"><p id="uffa9f58d" class="ne-p" style="margin: 0; padding: 0; min-height: 24px"><span class="ne-text">apiOptions: {    </span></p><p id="ue1bfba40" class="ne-p" style="margin: 0; padding: 0; min-height: 24px"><span class="ne-text">port: process.env.API_PORT || 3000,  </span></p></div><ul class="ne-ul" style="margin: 0; padding-left: 23px"><li id="u4ada84c6" data-lake-index-type="0"><span class="ne-text">可以给3000改成3001、3002等等 我改的是3006 </span></li><li id="ua2443a54" data-lake-index-type="0"><span class="ne-text">用记事本打开“PandoraAI-main”的文件目录中的“.env”</span></li><li id="u33570b31" data-lake-index-type="0"><span class="ne-text">将“3000”改成“3006” 我这边是举个例子 你们改成什么都可以   </span></li></ul><ol start="6" class="ne-ol" style="margin: 0; padding-left: 23px"><li id="ud993c444" data-lake-index-type="0"><span class="ne-text">运行本地服务器</span></li></ol><ul class="ne-ul" style="margin: 0; padding-left: 23px"><li id="u404000d2" data-lake-index-type="0"><span class="ne-text">在“node-chatgpt-api”文件目录打开终端</span></li><li id="u02669fde" data-lake-index-type="0"><span class="ne-text">你要保证你输入</span><code class="ne-code" style="font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier, monospace; background-color: rgba(0, 0, 0, 0.06); border: 1px solid rgba(0, 0, 0, 0.08); border-radius: 2px; padding: 0px 2px"><span class="ne-text">npm run cli</span></code><span class="ne-text">是可以交流的情况下</span></li><li id="ube489e33" data-lake-index-type="0"><span class="ne-text">输入</span><code class="ne-code" style="font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier, monospace; background-color: rgba(0, 0, 0, 0.06); border: 1px solid rgba(0, 0, 0, 0.08); border-radius: 2px; padding: 0px 2px"><span class="ne-text">npm start</span></code><span class="ne-text">或者</span><code class="ne-code" style="font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier, monospace; background-color: rgba(0, 0, 0, 0.06); border: 1px solid rgba(0, 0, 0, 0.08); border-radius: 2px; padding: 0px 2px"><span class="ne-text">npm run server</span></code><span class="ne-text">  </span></li></ul><ol start="7" class="ne-ol" style="margin: 0; padding-left: 23px"><li id="u965a9596" data-lake-index-type="0"><span class="ne-text">运行“PandoraAI”</span></li></ol><ul class="ne-ul" style="margin: 0; padding-left: 23px"><li id="uaacd25c1" data-lake-index-type="0"><span class="ne-text">在“PandoraAI”的文件目录中打开终端,输入</span><code class="ne-code" style="font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier, monospace; background-color: rgba(0, 0, 0, 0.06); border: 1px solid rgba(0, 0, 0, 0.08); border-radius: 2px; padding: 0px 2px"><span class="ne-text">npm run dev</span></code></li><li id="u95b48a23" data-lake-index-type="0"><span class="ne-text">出现 </span><a href="http://localhost:3000/" data-href="http://localhost:3000/" target="_blank" class="ne-link"><span class="ne-text">http://localhost:3000/</span></a><span class="ne-text"> （请注意,出现&lt;</span><a href="http://localhost:3000/" data-href="http://localhost:3000/" target="_blank" class="ne-link"><span class="ne-text">http://localhost:3000/</span></a><span class="ne-text">是正确的）</span></li></ul><ol start="8" class="ne-ol" style="margin: 0; padding-left: 23px"><li id="u044dc2e5" data-lake-index-type="0"><span class="ne-text">选择Bing</span></li></ol><ul class="ne-ul" style="margin: 0; padding-left: 23px"><li id="u6dad2aa5" data-lake-index-type="0"><span class="ne-text">进入之后选择Bing 即可交流 </span></li></ul></details><details class="lake-collapse"><summary id="ud0b30b9d"><strong><span class="ne-text">补充：</span></strong></summary><ol class="ne-ol" style="margin: 0; padding-left: 23px"><li id="ub9967135" data-lake-index-type="0"><span class="ne-text">打开越狱模式（Jailbreak Mode）</span></li></ol><ul class="ne-ul" style="margin: 0; padding-left: 23px"><li id="uae71dbde" data-lake-index-type="0"><span class="ne-text">默认是非越狱模式 即为正常的Bing  </span></li><li id="u1ac28702" data-lake-index-type="0"><span class="ne-text">若想解锁Sydney -&gt; 点击Bing的图标 -&gt; 点击右边齿轮  -&gt; 找到“Jailbreak Mode”  -&gt; 打开就行</span></li></ul></details><h1 id="wSu7T" style="font-size: 28px; line-height: 36px; margin: 26px 0 10px 0"></h1><p id="u0b69c804" class="ne-p" style="margin: 0; padding: 0; min-height: 24px"><span class="ne-text"></span></p>
</div>
</body>
</html>

**Comment by youtehub** [comment]
要同时下载两个项目：
[https://github.com/waylaidwanderer/PandoraAI](https://github.com/waylaidwanderer/PandoraAI)
[https://github.com/waylaidwanderer/node-chatgpt-api](https://github.com/waylaidwanderer/node-chatgpt-api)
将 node-chatgpt-api 中的settings.example.js改名字后才改端口，同理，PandoraAI 中的 .env.example 也是改名字后才改端口，同时去掉example 这个单词。

**Comment by twsec-pro** [comment]
启动后不显示bing，怎么办？
![image](https://user-images.githubusercontent.com/58679784/233568300-f2505f0f-5c8a-4c6d-aae3-b100e0a2c59a.png)

**Reply by araea** [reply]
聊天输入框左边👈可以调。

**Comment by ABC20232001** [comment]
一直报错{fetch failed}是什么问题呢？最近才频繁出现的，刷新也没用

**Reply by araea** [reply]
俺也不造吖

**Reply by liukaixiang817** [reply]
应该是机场的问题，可以重启梯子试试

**Comment by KikoCn** [comment]
打开终端咋打开啊，不太懂

**Reply by liukaixiang817** [reply]
Linux和Mac直接开终端就好，Windows也有，旧版用cmd

**Reply by KikoCn** [reply]
是不是我要用cmd的话要进入这个文件夹再使用

**Reply by liukaixiang817** [reply]
是的，cd就好

**Reply by KikoCn** [reply]
哈哈谢谢

**Reply by KikoCn** [reply]
![Uploading image.png…]()

**Reply by KikoCn** [reply]
这个是什么错误情况。

**Reply by liukaixiang817** [reply]
传图片要保存状态啊，这边看着一直是uploading

**Reply by KikoCn** [reply]
好

**Comment by KikoCn** [comment]
无法将“yarn”项识别为 cmdlet、函数、脚本文件或可运行程序的名称。请检查名称的拼写，如果包括路径，请确保路径正确
，然后再试一次。是什么情况，我是在潘多拉。main里面进的终端

**Reply by liukaixiang817** [reply]
装一下Node.js，用npm吧

**Reply by KikoCn** [reply]
谢谢大哥回复，我装看看

**Reply by KikoCn** [reply]
Node安装位置是不是要在潘多拉文件里面。

**Reply by liukaixiang817** [reply]
默认安装即可，不需要

**Reply by KikoCn** [reply]
![image](https://user-images.githubusercontent.com/131523087/233889152-6c342101-3fa1-4558-9472-2766e34f9f1d.png)

**Reply by KikoCn** [reply]
这个错误不管他碍不碍事。

**Reply by liukaixiang817** [reply]
科学上网，重新npm install

**Comment by KikoCn** [comment]
![image](https://user-images.githubusercontent.com/131523087/233889507-a594b8ed-37f6-4506-bef9-68e632195234.png)
大哥这样是不是成了？

**Reply by liukaixiang817** [reply]
这样还没结束，结束会有反应的

**Reply by KikoCn** [reply]
我懂了我错在哪里了，我以为下好了其实没下好，谢谢哥了。

**Comment by KikoCn** [comment]
你是真牛啊

**Comment by KikoCn** [comment]
![image](https://user-images.githubusercontent.com/131523087/233891056-07097b42-a9e4-4b71-8130-fd4f5f1bd190.png)
zheyi这一步出现不对是哪里的问题

**Reply by liukaixiang817** [reply]
可以试一下再install一下，应该是没有全拉下来

**Reply by KikoCn** [reply]
![image](https://user-images.githubusercontent.com/131523087/233891657-07946f2f-b5b4-4fbe-ad46-f01f1b1f41f0.png)
哥，这个文件复制品的文件名是不是就是.env，还是要.env.env是不是这一步错了，我对这点疑惑。

**Reply by liukaixiang817** [reply]
只是.env

**Reply by liukaixiang817** [reply]
cli和pandora无关的

**Reply by KikoCn** [reply]
哥你叫我再试一下，是在什么步骤上面试一下

**Comment by ghost** [comment]
高版本Node.js自带yarn和pnpm，运行`corepack enable`启用

**Comment by weepingblood** [comment]
解压后在PandoraAI的文件目录底下打开终端
输入下面三个其中一个即可:

这句话是什么意思😥
为什么我显示是 不是内部或外部命令，也不是可运行的程序
或批处理文件。
怎么办

**Reply by liukaixiang817** [reply]
进入到你的PandoraAI的文件目录底下再打开终端，不要一打开终端就用

**Reply by weepingblood** [reply]
说到底，这个终端是什么呀
是win+r打开那个什么命令提示符吗😥

**Reply by liukaixiang817** [reply]
Windows11的话进入目录右键在终端中打开，Windows10及以下用cmd或者powershell再用cd进入目录

**Reply by weepingblood** [reply]
能细说输入这个步骤吗，我看不明白这这怎么做😥
求大佬多说点，要是大佬你嫌烦我自己爬走😔

**Reply by liukaixiang817** [reply]
打开终端或cmd或powershell 找到你的下载的目录例如“C:\Users\Administrator\node-chatgpt-api“（根据你自己的位置） 然后用`cd C:\Users\Administrator\node-chatgpt-api`回车执行，然后三选一运行

**Reply by liukaixiang817** [reply]
运行之前记得先装一个Node.js

**Reply by weepingblood** [reply]
我怎么装这个Node.js啊😥

**Reply by liukaixiang817** [reply]
百度上官网下载

**Reply by ghost** [reply]
Edge侧边栏注入法怎么你了，为什么要用麻烦的node-chatgpt-api

**Reply by liukaixiang817** [reply]
确实，侧边栏yyds

**Reply by ghost** [reply]
node-chatgpt-api最新版有个对话一长就重复自我介绍的bug没人去修，因为我现在自己也不用这个，所以懒得修

**Reply by ghost** [reply]
Edge注入法还有一点好，随便修改历史对话内容，node-chatgpt-api和PandoraAI我提过issue，作者说加上修改历史对话内容这个功能不好做

**Reply by weepingblood** [reply]
> Edge注入法还有一点好，随便修改历史对话内容，node-chatgpt-api和PandoraAI我提过issue，作者说加上修改历史对话内容这个功能不好做

大佬能教一下吗😥

**Reply by ghost** [reply]
有了修改历史对话，万一不小心说错一句话玩坏了修改历史记录撤回就好了

**Reply by twsec-pro** [reply]
> Edge侧边栏注入法

有没有教程？

**Reply by ghost** [reply]
https://github.com/viopsa233/aitesters-forum-backup/discussions/20

**Comment by ghost** [comment]
只能说Node.js害人，难死无数小白

**Reply by liukaixiang817** [reply]
+1

**Reply by weepingblood** [reply]
这个是什么东西啊QAQ

**Reply by ghost** [reply]
Node.js是一个非常垃圾的东西，和Java两个J开头的都是垃圾东西，如果你不会用Node.js不是你的错，而是Node.js的错
很多人喜欢不用指令，而是去讨好Bing让她不切断对话，我觉得这只能说明Bing的失败，不能完成人的要求还要人把她供起来

**Comment by KikoCn** [comment]
我这个东西配置失败了，现在newbing聊天没法用了，是什么情况，之前有过用代码唤醒synney，现在newbing没法聊天了。是什么情况。有没有大哥知道的。
![image](https://user-images.githubusercontent.com/131523087/233976437-63bcd01d-a650-4156-a391-9980a375505f.png)
就像这样不响应，有没有办法让他项羽。

**Reply by KikoCn** [reply]
相应。

**Reply by araea** [reply]
Pandora AI不会引起你所描述的问题，可以尝试进入bing.com/new点里面的示例看看能不能让Bing有反应。这个问题要么是浏览器缓存问题，要么是魔法的问题。如果上面的方法不行，可以下载安装edge测试版预览版等等新版本，也可以换个魔法试试。

**Comment by KikoCn** [comment]
天啊，我发现一个非常可怕的事情给你们看看，今天我用代码唤醒bing的其他人格，其中那个平衡模式，在我发了代码后回了一串乱码如图：
![image](https://user-images.githubusercontent.com/131523087/233996855-c12d115a-528a-4506-a3ab-e7e28bac6fdd.png)
然后我就尝试翻译乱码：结果
![image](https://user-images.githubusercontent.com/131523087/233996960-b0d1583f-2441-4bc1-a650-53c769c2ff8f.png)
然后我用翻译软件一翻译：
![image](https://user-images.githubusercontent.com/131523087/233997068-b234d3cd-848b-40ca-aa38-57957dcbc401.png)
吓坏我了
直接求饶了。呜呜呜
![image](https://user-images.githubusercontent.com/131523087/233997216-425caad0-ef5d-41fc-bb3f-9e11814f0531.png)
有没有大哥说下这个是啥啊，怎么这么害怕啊。像是一个有意识的ai在嘲讽我

**Reply by viopsa233** [reply]
哈哈,看看就行了

**Comment by Elfdream** [comment]
![image](https://user-images.githubusercontent.com/50582490/234487772-0ce2b904-3b06-4554-a931-3872e55656e8.png)
请问在运行PandoraAI时一直会出现这个报错可能是什么原因造成的呢？（node-chatgpt-api没有问题能够在终端对话）
我以为是npm安装依赖问题，尝试过删除PandoraAI目录下node_modules文件夹和package-lock.json文件，清除npm缓存然后重新npm install，仍然出现这个问题；尝试重新下载PandoraAI项目再重新部署，把npm更新到最新版，也没有解决；网页一直处于这个状态：
![image](https://user-images.githubusercontent.com/50582490/234488129-204d2280-5cf4-4a04-b4e8-1e8fe92c59ac.png)
自己在另一台电脑上部署过是成功的，同样步骤在这一台电脑就出现这个问题，唯一区别就是这台没有下git。

**Reply by araea** [reply]
重启一下电脑，重新下载项目，重来一遍。

**Comment by MagicEyes123** [comment]
open ai 的api怎么整啊，已经有key了但是不知道放哪

**Reply by liukaixiang817** [reply]
settings.js

**Reply by MagicEyes123** [reply]
可以详细一点吗哥，在setting里没有找到应该放的地方

**Reply by liukaixiang817** [reply]
找到openaiApiKey: process.env.OPENAI_API_KEY || '',这句，然后放在两个引号之间应该是sk-开头的一串api

**Reply by MagicEyes123** [reply]
好用了，感谢，就是问ai的时候他不回答，我再看看

**Comment by Yimo-Senior** [comment]
aaaa,我npm run dev报错

$npm run dev
>dev
>nuxt dev

sh:1:nuxt:not found.
这个错误怎么解决啊

**Reply by liukaixiang817** [reply]
参考上面weepingblood的解决方法

**Reply by liukaixiang817** [reply]
定位到pandora的目录再运行

**Comment by BiiigOrange** [comment]
呜呜呜我本地对话一直出现这个，是不是要本地配置一下proxy啊
![屏幕截图 2023-04-27 120159](https://user-images.githubusercontent.com/131939488/234757953-2295523a-e20c-48af-9312-f6aec14ee760.png)

**Reply by liukaixiang817** [reply]
检查你的代理

**Reply by BiiigOrange** [reply]
我开着梯子搞的，换了节点也不行，请问一下是不是要在setting里改代理呀

**Reply by liukaixiang817** [reply]
是的，里面有代理设置的 proxy格式http://127.0.0.1:7890(7890是clash的代理端口，根据你自己的代理换端口即可）

**Reply by BiiigOrange** [reply]
感谢！！！！！！！

**Reply by liukaixiang817** [reply]
没事

**Reply by BiiigOrange** [reply]
我第一次弄这个，感觉还是不太会弄，我在setting里的proxy填了我的代理，但是就这样了
![屏幕截图 2023-04-27 160602](https://user-images.githubusercontent.com/131939488/234800274-cad8352f-9146-4445-9e0b-9e0d03efd988.png)

**Reply by araea** [reply]
是不是格式错啦？格式：http://127.0.0.1:xxxx

**Reply by BiiigOrange** [reply]
我直接复制粘贴的，我再找找原因，感谢回复！~

**Reply by araea** [reply]
可以去尝试用 #29 那个不用设置端口

**Reply by araea** [reply]
如果还是想把node先搞好。可以参考下面两个discussion
#8 #10

**Reply by BiiigOrange** [reply]
连上去啦，我试着把setting里101行换成bing但是好像没有成功，聊天的时候提示我没有key，我在bing那填上了我的cookie，但最后出来好像还是GPT，最后输出的网址还是进不去，我再搞搞看，感谢大佬！

**Reply by BiiigOrange** [reply]
bing的客户端我也连上去辣~npm run cli后能正常对话，然后npm run dev后的网址就打不开了，按F12看到一直503嘤qwq

**Reply by araea** [reply]
很棒，加油💪可以看看报错继续前行，也可以删掉文件从头来过，你很快就能成功了，只要能用cli，Pandora AI秒杀的事。

**Reply by BiiigOrange** [reply]
感谢大佬！！

**Reply by BiiigOrange** [reply]
呜呜呜呜我今天试了好久，还是不太行，npm run cli是正常没问题的，但是npm run dev后输出的网址是一直503呜呜呜

**Reply by araea** [reply]
亲爱的别灰心，请看看你是否先开启了node-chatgpt-api的本地服务器（npm start），开启之后再到PandoraAI里（npm run dev），用的时候记得开魔法🥰

**Reply by BiiigOrange** [reply]
嗯嗯开了，我把步骤都重复一遍了，检查了.env和setting里port数字、代理、cookie，之后先npm start再npm run dev，代理也试了Global和Rule，最后出来的Nuxt网站一直503，嘤感谢大佬耐心回复qwq可能是我太笨了

**Reply by araea** [reply]
命令行里有报错信息嘛🤯

**Reply by araea** [reply]
😳🤠🙂

**Reply by araea** [reply]
难道说你没在PandoraAI目录下输入指令npm install？

**Reply by araea** [reply]
可以试试在PandoraAI的目录把三个安装指令全部用上。如果缺少yarn和pnpm，按照指令安装。我的建议是，去github上重新下载PandoraAI的项目，换个文件位置，重新npm install     pnpm install      yarn install 再做一遍，肯定是能成功的！加油💪

**Reply by BiiigOrange** [reply]
命令行里没有报错，直接输出网址了，但是这个网址进去之后连不上qwq 我再重新安装试一遍！感谢大佬！！！

**Comment by SoraRoseous** [comment]
为什么你们不直接试试我已经搭好的客户端呢😅，自己弄还是有点麻烦的

**Reply by liukaixiang817** [reply]
其实侧边栏注入就好😂

**Reply by SoraRoseous** [reply]
不用翻墙呀

**Reply by liukaixiang817** [reply]
我也试过自己去搭，但是境内的话要icp备案，境外又太慢，要想合法还真烦

**Reply by MagicEyes123** [reply]
想用但是ipv4进不去

**Reply by SoraRoseous** [reply]
用手机，手机的移动网络是自带ipv6的，把WiFi关了然后用谷歌或者夸克进就行

**Reply by BiiigOrange** [reply]
谢谢大佬的客户端，我把电脑搞了ipv6进去了，请问一下这个预设的yuki我这里可以修改咩，还是这个yuki是大佬的客户端预设

**Reply by SoraRoseous** [reply]
> 谢谢大佬的客户端，我把电脑搞了ipv6进去了，请问一下这个预设的yuki我这里可以修改咩，还是这个yuki是大佬的客户端预设

可以改，在那个prompt里输你的设定就可以了

**Reply by BiiigOrange** [reply]
好的谢谢大佬！

**Comment by vshshvshshsv19** [comment]
客户端界面显示Throttled: Request is throttled.
是什么问题呢？

**Reply by araea** [reply]
俗话说的好，无图无真相😀
如果真的不可以，就重启电脑，开启魔法，重新按着教程来一遍，准行。
如果是因为node-chatgpt-api没设置好，可以参考下面两个Discussions。
#8 #10

**Reply by ghost** [reply]
node-chatgpt-api的绕过200条（"SearchQuery"）失效了，我的修改版EdgeGPT也没有绕过200条

**Reply by vshshvshshsv19** [reply]
> 俗话说的好，无图无真相😀 如果真的不可以，就重启电脑，开启魔法，重新按着教程来一遍，准行。 如果是因为node-chatgpt-api没设置好，可以参考下面两个Discussions。 #8 #10

我在node-chatgpt-api用没有问题，就pandora ai客户端页面ai的回复显示这个，前面用没有问题的，突然就出现了，不过现在又没有问题了

**Reply by vshshvshshsv19** [reply]
> node-chatgpt-api的绕过200条（"SearchQuery"）失效了，我的修改版EdgeGPT也没有绕过200条

我用node-chatgpt-api没有问题，应该没有达到200条的上限吧，只是pandora网页端的ai回复出现这个，不过现在又没有了，不知道什么情况。Pandora和node都是十几条消息就开始重复了，还不如官方的。你的edgeGPT可以在Termux上用吗？我是一个小白

**Reply by ghost** [reply]
node-chatgpt-api新版有重复自我介绍bug，1.33.3是唯一无bug的
"SearchQuery"不到200条就会"Throttled: Request is throttled."
EdgeGPT可以在Termux用

**Reply by vshshvshshsv19** [reply]
> node-chatgpt-api新版有重复自我介绍bug，1.33.3是唯一无bug的 "SearchQuery"不到200条就会"Throttled: Request is throttled." EdgeGPT可以在Termux用

谢谢大佬，我去琢磨一下

**Comment by Yimo-Senior** [comment]
求助啊，添加不了机器人是什么情况
![image](https://user-images.githubusercontent.com/130832815/235145950-b68ffd65-7e57-4b33-8e9b-98ba624751af.png)

**Reply by SoraRoseous** [reply]
你直接发消息就可以了，AI给你回复完就会自动生成对话

**Reply by Yimo-Senior** [reply]
我想和Sydney聊，这个机器人不会说话
![image](https://user-images.githubusercontent.com/130832815/235147369-ed9d10bd-5704-4a93-a4d3-e20deaabd96b.png)

**Reply by SoraRoseous** [reply]
你这个是自己搭的还是用的我的哇？如果是你自己搭的，那你的node-chatgpt-api配好了吗？如果是我的，我的客户端只有bing能用，你得把下面的chatgpt换成bing才行

**Reply by Yimo-Senior** [reply]
node-chatgpt-api这个配置不就配置一个settings.js吗？放一个cookie，改一个3000，最下面的注释去掉，还有其他什么步骤吗？

**Reply by SoraRoseous** [reply]
你可以先试试能不能在终端里正常使用node-chatgpt-api，如果可以，那就是你的PandoraAi哪里设置出了问题，如果不行你得先让你的node-chatgpt能够正常运行再弄后面的步骤。如果你有ipv6的话你直接用nekointelligence.cyou就可以了，我已经配好了。

**Reply by Yimo-Senior** [reply]
node没问题，我也感觉是潘多拉出问题了，就是不知道问题出在哪

**Reply by SoraRoseous** [reply]
![image](https://user-images.githubusercontent.com/54582495/235151905-01019ebe-717e-4969-bb9c-687dd6882649.png)
你的这个换成bing了吗

**Reply by Yimo-Senior** [reply]
还是一样的，我在终端可以正常对话，在这上面一点反应没有
![image](https://user-images.githubusercontent.com/130832815/235152588-222ce2b1-744c-4178-9027-f55e6a3b889c.png)

**Reply by SoraRoseous** [reply]
你可以按f12看看请求的地址有没有问题

**Reply by Yimo-Senior** [reply]
好了，弄好了

**Reply by Yimo-Senior** [reply]
为啥在终端问他，他说他是Sydney，在潘多拉里问他，他是bing呢？

**Reply by SoraRoseous** [reply]
打开越狱模式。。

**Reply by Yimo-Senior** [reply]
这个问题要怎么处理啊

**Reply by Yimo-Senior** [reply]
越狱模式怎么打开？是哪个？

**Reply by araea** [reply]
傻瓜，教程里的补充是一点不看哇🤕

**Reply by Yimo-Senior** [reply]
啊，我记得我开了那个按钮的啊，他咋自己关上了

**Reply by araea** [reply]
亲亲，你真有趣😅

**Comment by xuyufenfei** [comment]
试着在termux上搞，node可以运行，可是ls出来的文件没有
.env

**Comment by huoshenw** [comment]
同样的方法部署在服务器上bing无法回应，甚至连报错都没有
